ingest-app/test/cmr/ingest/services/granule_bulk_update/s3/s3_util_test.clj

```diff
@@ -0,0 +1,23 @@
+(ns cmr.ingest.services.granule-bulk-update.s3.s3-util-test
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.ingest.services.granule-bulk-update.s3.s3-util :as s3-util]))
+
+(deftest validate-url-test
+  (testing "validate url"
+    (is (= ["s3://abc/foo" "S3://abc/bar"]
+           (s3-util/validate-url "s3://abc/foo, S3://abc/bar"))))
+
+  (testing "validate url error scenarios"
+    (are3 [url-value re]
+      (is (thrown-with-msg?
+           Exception re (s3-util/validate-url url-value)))
+
+      "invalid s3 link"
+      "http://example.com/foo"
+      #"Invalid URL value, each S3 url must start with s3://, but was http://example.com/foo"
+      
+      "nvalid s3 link in multiple urls"
+      "s3://abc/foo,http://example.com/bar,s3://abc/baz"
```

Comment:
We should add a case where the `S` in `s3` is capitalized

Reply:
I have an active question in the granule bulk update channel to limit the S3 link to be only `s3://`, not `S3://`. I will change one of the s3 to S3 if we decide to allow both.

Reply:
Actually, the `S3` is already checked on line 9 in the same file. No need to do it here again.